init.sh

```diff
@@ -18,7 +18,7 @@ sudo apt-get install -y --force-yes \
     build-essential python-dev \
     python-pip python-numpy \
     python-nose python-nosexcover pylint pep8 \
-    python-gdal python-opencv
+    python-gdal
```

Comment:
are you really sure nothing else uses opencv?  I'm not convinced.  I'd lead this in unless you are really certain.